### DISCUSSION TITLE
Get shades of a tone

### DISCUSSION THREAD

**Discussion by jondashkyle** [mirror-xyz/degen#52] [Q&A]
A common pattern I’ve found is passing a `tone` as a prop to a component, then within that component wanting to grab a certain shade. For example:

```
<FooBar tone="blue" />
```

Then say within the component I’d like to use the `tertiary` shade of the tone. What’s the best practice on how to grab that programmatically?

cc @tmm

**Comment by metalica90** [comment]
Nice
